<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-07-17T21:16:03 IST" name="Suite" finished-at="2021-07-17T21:17:16 IST" duration-ms="73170">
    <groups>
    </groups>
    <test started-at="2021-07-17T21:16:03 IST" name="Test" finished-at="2021-07-17T21:17:16 IST" duration-ms="73170">
      <class name="com.crm.tests.LoginTest">
        <test-method is-config="true" signature="intialize()[pri:0, instance:com.crm.tests.LoginTest@62230c58]" started-at="2021-07-17T21:16:03 IST" name="intialize" finished-at="2021-07-17T21:16:07 IST" duration-ms="4108" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- intialize -->
        <test-method signature="logintest()[pri:1, instance:com.crm.tests.LoginTest@62230c58]" started-at="2021-07-17T21:16:07 IST" name="logintest" finished-at="2021-07-17T21:16:14 IST" duration-ms="6685" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logintest -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.crm.tests.LoginTest@62230c58]" started-at="2021-07-17T21:16:14 IST" name="teardown" finished-at="2021-07-17T21:16:14 IST" duration-ms="138" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
        <test-method is-config="true" signature="intialize()[pri:0, instance:com.crm.tests.LoginTest@62230c58]" started-at="2021-07-17T21:16:14 IST" name="intialize" finished-at="2021-07-17T21:16:17 IST" duration-ms="3477" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- intialize -->
        <test-method signature="invalidlogintest()[pri:2, instance:com.crm.tests.LoginTest@62230c58]" started-at="2021-07-17T21:16:17 IST" name="invalidlogintest" finished-at="2021-07-17T21:16:27 IST" duration-ms="9656" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Invalid login1] but found [Invalid login]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Invalid login1] but found [Invalid login]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.crm.tests.LoginTest.invalidlogintest(LoginTest.java:52)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidlogintest -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.crm.tests.LoginTest@62230c58]" started-at="2021-07-17T21:16:27 IST" name="teardown" finished-at="2021-07-17T21:16:27 IST" duration-ms="118" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.crm.tests.LoginTest -->
      <class name="com.crm.tests.FormsTest">
        <test-method is-config="true" signature="intialize()[pri:0, instance:com.crm.tests.FormsTest@2df9b86]" started-at="2021-07-17T21:16:49 IST" name="intialize" finished-at="2021-07-17T21:16:52 IST" duration-ms="3424" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- intialize -->
        <test-method signature="formTest()[pri:0, instance:com.crm.tests.FormsTest@2df9b86]" started-at="2021-07-17T21:16:52 IST" name="formTest" finished-at="2021-07-17T21:17:16 IST" duration-ms="23456" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- formTest -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.crm.tests.FormsTest@2df9b86]" started-at="2021-07-17T21:17:16 IST" name="teardown" finished-at="2021-07-17T21:17:16 IST" duration-ms="198" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.crm.tests.FormsTest -->
      <class name="com.crm.tests.ContactsTest">
        <test-method is-config="true" signature="intialize()[pri:0, instance:com.crm.tests.ContactsTest@793be5ca]" started-at="2021-07-17T21:16:27 IST" name="intialize" finished-at="2021-07-17T21:16:31 IST" duration-ms="3553" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- intialize -->
        <test-method signature="contactsTest()[pri:0, instance:com.crm.tests.ContactsTest@793be5ca]" started-at="2021-07-17T21:16:31 IST" name="contactsTest" finished-at="2021-07-17T21:16:49 IST" duration-ms="18217" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- contactsTest -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.crm.tests.ContactsTest@793be5ca]" started-at="2021-07-17T21:16:49 IST" name="teardown" finished-at="2021-07-17T21:16:49 IST" duration-ms="87" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.crm.tests.ContactsTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
